caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -38,7 +38,7 @@ class ChartContainer extends React.Component {
 
       width: () => this.chartContainerRef.getBoundingClientRect().width,
 
-      height: () => parseInt(this.props.height, 10) - 100,
+      height: () => parseInt(this.props.height, 10),
```

Comment:
`- 100` here was because the bottom of some charts were being cut off.  probably a better way to handle this is in the margin declarations for each chart, or in the <ChartContainer /> component more specifically. 

can you check that the other charts are rendered correctly with this change?